src/main/assembly/filter.properties

```diff
@@ -0,0 +1,3 @@
+product.name=WSO2 ESB
```

Comment:
Even though this file is not used, shall we update these files with MI related values?

Reply:
Sure changed with [33fafe7](https://github.com/wso2-extensions/esb-connector-adobecampaignclassic/pull/1/commits/33fafe7b3b159683b566791f32a73c0bc28127d3).